src/Microsoft.ML.Core/Utilities/VBufferUtils.cs

```diff
@@ -625,9 +625,9 @@ public static void CreateMaybeSparseCopy<T>(ref VBuffer<T> src, ref VBuffer<T> d
         /// <param name="dst">Argument vector, whose elements are only read</param>
         /// <param name="res">Result vector</param>
         /// <param name="manip">Function to apply to each pair of elements</param>
-        public static void ApplyWithEitherDefinedCopy<TSrc, TDst>(ref VBuffer<TSrc> src, ref VBuffer<TDst> dst, ref VBuffer<TDst> res, PairManipulatorCopy<TSrc, TDst> manip)
+        public static void ApplyWithEitherDefinedCopy<TSrc, TDst>(in VBuffer<TSrc> src, ref VBuffer<TDst> dst, ref VBuffer<TDst> res, PairManipulatorCopy<TSrc, TDst> manip)
```

Comment:
>ref [](start = 88, length = 3)

/// <param name="dst">Argument vector, whose elements are only read</param>

Reply:
I will update this comment, since it is lying.

---
In reply to: [229498807](https://github.com/dotnet/machinelearning/pull/1454#discussion_r229498807) [](ancestors = 229498807)